src/cromshell/status/command.py

```diff
@@ -0,0 +1,144 @@
+import fileinput
+import json
+import logging
+
+import click
+import requests
+import csv
+from typing_extensions import Counter
+
+from cromshell.utilities import cromshellconfig
+from cromshell.utilities import http_utils
+from cromshell.utilities import io_utils
+from cromshell import log
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="status")
+@click.argument("workflow_id")
+@click.pass_obj
+def main(config, workflow_id):
+    """Check the status of a Cromwell job UUID"""
+
+    LOGGER.info("status")
+
+    ret_val = 0
+
+    # Set cromwell server using submission file. Running the function below with
+    # passing only the workflow id overrides the default cromwell url set in the
+    # cromshell config file, command line argument, and environment. This takes
+    # place only if the workflow id is found in the submission file.
+    cromshellconfig.resolve_cromwell_config_server_address(
+        workflow_id=workflow_id
+    )
+
+    # Check if Cromwell Server Backend works
+    http_utils.assert_can_communicate_with_server(config)
+
+    # Request workflow status
+    request_out = requests.get(
+        f"{config.cromwell_server}{config.api_string}{workflow_id}/status"
```

Comment:
Since `{config.cromwell_server}{config.api_string}{workflow_id}` is used in a lot of places, you can define a variable for this somewhere (probably in the config) and reference it with a single variable instead of 3.

Reply:
The workflow_id is a Status command argument so I wont be able to set this variable in main.py. The config module is initiated in the main.py so if i try to concatenate {config.cromwell_server}{config.api_string}{workflow_id} in the module then the code fails because workflow Id hasn't been set. 
I ended up creating a variable called `cromwell_api_workflow_id` in the config and set that to None, it will be set in the status subcommand